pkg/credentialprovider/aws/aws_credentials.go

```diff
@@ -34,6 +34,7 @@ import (
 // and credentialprovider.
 var AWSRegions = [...]string{
 	"us-east-1",
+	"us-east-2",
```

Comment:
It's been merged already ... surprised this isn't a conflict.

https://github.com/kubernetes/kubernetes/blob/master/pkg/credentialprovider/aws/aws_credentials.go

@StudyNick it's probably a good idea to do one PR per "idea" - even if in this case it means 3 PRs.  Not least because then I could have merged your us-east-2 PR and it would have gone into k8s sooner ;-)